### ISSUE TITLE
Table reference to diverge in workers node after adding new column

### ISSUE THREAD

**Issue by levanhai82** [citusdata/citus#5474]
Hi Team,

Our Citus database have 1 coordinate and 2 workers node. 
Table reference "transaction.b2c_project" have entire content is replicated in workers node.
After we add the new **detail_path** column, data value of column **detail_path** to diverge in worker node. 

The application code uses below query for inserting and updating.
```
INSERT INTO transaction.b2c_project (b2c_project_id, detail_path, project_code, project_name, property_type_id)
VALUES ($1, $2, $3, $4, $5)
```
```
UPDATE transaction.b2c_project SET detail_path = $1, project_name = $2
WHERE b2c_project_id = $3;
```

What do we need to do after adding new columns to the table reference for replicating them in workers node?
 
Thank you.

**Comment by onurctirtir** [comment]
I believe this is same as https://github.com/citusdata/citus/issues/5424, right ?
As we discussed via your ticket on Azure Support Center, you might want to re-sync those diverged values by first pulling data to coordinator, then re-replicating your table to worker nodes, as in:
```sql
SELECT undistribute_table('<table_name>');
SELECT create_reference_table('<table_name>');
```

The node that the first command (`undistribute_table`) would choose to the read the data from would be the same node that the `UPDATE` command that you have run before based on tuple `xmin` values, as long as you didn't set `task_assignment_policy` to a non-default value **when running those updates**.

Another important point is that, `undistribute_table` is introduced in Citus 9.5, so you need to be using 9.5 to make use of the manual fix.

**Comment by levanhai82** [comment]
Hi @onurctirtir, We clarify more about this issue after troubleshooting period. We have two reference tables A and B.
Firstly, table A has data issue as [#5424](https://github.com/citusdata/citus/issues/5424),
Then the script/application pulls data directly from tables A to update/insert for tables B. So table B also has data issue.
We have fixed dev code for  [#5424](https://github.com/citusdata/citus/issues/5424) and manual fix data of reference table. 
We will follow and monitor this issue in several days.
